app/celer-dump-data.cc

```diff
@@ -968,7 +968,7 @@ void print_optical_materials(std::vector<ImportOpticalModel> const& io_models,
 /*!
  * Execute and run.
  */
-int main(int argc, char* argv[])
+int main(int argc, char* argv[])  // NOLINT(bugprone-exception-escape)
```

Comment:
Better would be to move the main body to a `run` function and wrap with try/catch I think... this is fine for now though.